### PR TITLE
add: command button for small screen device

### PR DESCRIPTION
**Pull request by hemik000**
- added a command button for the small screen devices.
- button will be hidden on the print page
#24

### CODE REVIEW

**Comment by vktechturbine** [comment]
you want add command button for small screen like mobile to print your cv is that right
please let me know

**Review by OmitNomis — APPROVED** [review]
Looks good, nice change 👍🏽

File: src/components/command-menu.tsx [hunk 1]
```diff
@@ -33,13 +35,21 @@ export const CommandMenu = ({ links }: Props) => {
...
         Press{" "}
         <kbd className="pointer-events-none inline-flex h-5 select-none items-center gap-1 rounded border bg-muted px-1.5 font-mono text-[10px] font-medium text-muted-foreground opacity-100">
           <span className="text-xs">⌘</span>J
         </kbd>{" "}
         to open the command menu
       </p>
+      <Button
+        onClick={() => setOpen((open) => !open)}
+        variant={"outline"}
+        size={"icon"}
```
Comment (BartoszJarocki) [inline src/components/command-menu.tsx:48]: to be consistent with rest of the props lets not use `{}`

**Review by BartoszJarocki — APPROVED** [review]
just one little thing and it's good to go! nice change

File: src/components/command-menu.tsx (hunk 1 above)
Comment (hemik000) [reply src/components/command-menu.tsx]: done

**Review by OmitNomis — APPROVED** [review]

**Pull request merged**